Module2/springhometask/src/main/java/com/epam/spring/model/UserAccount.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.spring.model;
+
+
+public interface UserAccount {
```

Comment:
not used anywhere

Reply:
I've created this in order to use in `UserAccountMapper` to implement future functionality

Reply:
but it should be implemented by UserAccountEntity?

Reply:
agree! thanks for mentioning